app/main.py

```diff
@@ -2,4 +2,57 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords if coords else [0, 0]
```

Comment:
so you don't need here `else [0, 0]` if you specify it in annotation
```suggestion
        self.coords = coords
```

Reply:
if I fix the code as you indicated, the tests will fail